dscp-node/test/helper/logger.js

```diff
@@ -0,0 +1,21 @@
+const pino = require('pino')
+
+const logger = pino(
+  {
+    name: 'API',
+    level: process.env.LOG_LEVEL,
+    redact: {
+      paths: ['USER_URI', '[*].USER_URI'],
+      censor: (args) => {
+        if (args === '' || args === null || args === undefined) {
+          return '[EMPTY]'
+        } else {
+          return '[REDACTED]'
+        }
+      },
+    },
+  },
+  process.stdout
+)
+
+module.exports = logger
```

Comment:
why do we need a mock logger?

Reply:
to log during tests. If we're moving to exporting the whole api though I'll add the logger to `src/` and have its log level configurable as well